### DISCUSSION TITLE
Cloud File managers as a new site type

### DISCUSSION THREAD

**Discussion by mahemmati** [vitodeploy/vito#223] [Ideas]
Hi there Saeed!
It would be very handy if we could install "File explorer/manager" web apps such as https://github.com/filebrowser/filebrowser or https://github.com/filegator/filegator as a site type with Vito.

**Comment by saeedvaziry** [comment]
@mahemmati as I also explained [here](https://vitodeploy.featurebase.app/p/install-a-filemanager-via-vitodeploy/comment/6640d3f67873d01b709b0a3e), you can install a filemanager like Tinyfilemanager as a PHP site which is already supported.

Adding more site types like filemanagers are not planned on Vito.

For filemanagers that are written in other languages like Golang we need reverse proxies which is not supported on Vito at the moment.

However it is planned to support other languages on VitoDeploy  version 2 which will enable you to also install filemanagers like the one you mentioned.
